ADR/0013-integration-service-api-contracts.md

```diff
@@ -93,11 +97,27 @@ Example for a run with an error:
     "result": "ERROR",
     "timestamp": "1649842004",
     "successes": 0,
+    "note": "Task fbc-validation failed: $(workspaces.source.path)/hacbs/inspect-image/image_inspect.json did not generate correctly. For details, check Tekton task result HACBS_TEST_OUTPUT in task inspect-image.",
     "failures": 0,
     "warnings": 0
 }
 ```
 
+
+#### Tekton Result Format for `CLAIR_SCAN_RESULT`
+The Test output of the Tekton result **CLAIR_SCAN_RESULT** will be a JSON object that includes the context about the following information about the found vulnerabilities scanned by Clair:
+
```

Comment:
Maybe we can mention the Red Hat Vulnerability documentation here (https://access.redhat.com/articles/red_hat_vulnerability_tutorial) to provide more context on the vulnerability severity ratings.

Reply:
Yes, good point. Thanks!

Reply:
Updated.

Reply:
Thanks, looks good.